install/kubernetes/helm/istio/values.yaml

```diff
@@ -108,6 +108,14 @@ global:
   # Whether to perform server-side validation of configuration.
   configValidation: true
 
+  # If set to true, the pilot and citadel mtls will be exposed on the
+  # ingress gateway
+  meshExpansion: false
+
+  # If set to true, the pilot and citadel mtls and the plain text pilot ports
+  # will be exposed on an internal gateway
+  meshExpansionILB: false
+
```

Comment:
You don’t need these two.

Reply:
Why not ? How else can user control if they add or not the gateway (and expose pilot/ca) ?